healthcheck/README.md

```diff
@@ -0,0 +1,10 @@
+## Healthcheck
+
+This service monitors the status of Zimfarm services and components and displays results as HTML
+
+### Environment variables
+
+- ZIMFARM_API_URL: Zimfarm backend API URL
+- ZIMFARM_USERNAME: Username to authenticate with on Zimfarm
+- ZIMFARM_PASSWORD: Password of user to authenticate with on Zimfarm
+- ZIMFARM_DATABASE_URL: Zimfam database URL
```

Comment:
`Zimfam` => typo + specify this should be full database URL including credentials